src/Form/Form.js

```diff
@@ -392,8 +392,7 @@ class Form {
     shouldOpenTooltip (e, input) {
         if (this.device.globalConfig.isApp) return true
 
-        const inputType = getInputMainType(input)
-        return (!this.touched.has(input) && this.areAllInputsEmpty(inputType)) || isEventWithinDax(e, input)
+        return (!this.touched.has(input) && !input.classList.contains('ddg-autofilled')) || isEventWithinDax(e, input)
```

Comment:
This was old logic for which `areAllInputsEmpty` was equivalent to the field having being autofilled, but now we have many more fields and we should check explicitly for `ddg-autofilled`.